signer/src/testing/mod.rs

```diff
@@ -112,3 +116,13 @@ where
 }
 
 impl<I, T> IterTestExt<T> for I where I: IntoIterator<Item = T> + Sized {}
+
+/// Returns a seedable rng with random seed. Prints the seed to
+/// stderr so that it can be used to reproduce the test
+pub fn get_rng() -> StdRng {
```

Comment:
Just a thought, maybe this should return `impl RngCore + Clone + Send` instead of a concrete type? I'm pretty sure we only need `RngCore` functionality in the tests themselves and it just helps keep things more encapsulated.

Reply:
Actually, we need `impl CryptoRng + RngCore + Clone + Send` here.

`StdRng` implements also `SeedableRng` `Eq` and `PartialEq`. I'm not 100% sure if it will help us, even if we will try to switch to another rng once, but let's do it

Reply:
I wouldn't include any constraints that we don't explicitly use in tests, anyway. But yeah, this method then becomes living documentation of what traits we're actually using as well.

Reply:
[fe63eb3](https://github.com/stacks-sbtc/sbtc/pull/1578/commits/fe63eb3f3b1ffae6eb094f312fcb80f6c35aa877)

Reply:
Nice, looks like you just missed a `cargo fmt` or something based on CI though